### ISSUE TITLE
User experience related changes for "addon" command

### ISSUE THREAD

**Issue by budhrg** [minishift/minishift#652]
While trying the addon command, I found a bit lack of user experience here.

- Some timing info when applying addon during `minishift start`:
```
$ minishift start       
Starting local OpenShift cluster using 'kvm' hypervisor...
-- Checking OpenShift client ... OK
-- Checking Docker client ... OK
-- Checking Docker version ... OK
.....
Applying addon cluster-admin:
-- adm policy add-role-to-user cluster-admin developer
role "cluster-admin" added: "developer"
****************************
  [160.958846ms]                    # What is it? 
****************************
```

- No message while enabling/disabling addon, I need to run `list` command to see the status:
```
# Expected
$minishift addons enable anyuid
Addon 'anyuid' enabled

$ minishift addons disable anyuid
Addon 'anyuid' disabled 
```
- No message to give pointer to how to use addon after install:
```
$ minishift addons install addons/expose-registry
Addon registry installed.

# Expected
$ minishift addons install addons/expose-registry 
Addon 'registry' installed. You need to enable it to use it.    # Note the single quote in 'registry'
```

**Comment by praveenkumar** [comment]
```
role "cluster-admin" added: "developer"
****************************
  [160.958846ms]                    # What is it? 
****************************
```

@budhrg This is the time taken for a command which run, it here to check the performance and time consumed. I think it's helpful (atleast to deve member).

**Comment by budhrg** [comment]
> This is the time taken for a command which run, it here to check the performance and time consumed. I think it's helpful (atleast to dev member)

@praveenkumar , if it is like that I would expect that for every command right? why this one only. 

I think this is kind of irrelevant information here for general users. We actually need a flag to enable such info for dev users. This can be done as part of `--verbose` flag as in this issue https://github.com/minishift/minishift/issues/634.

**Comment by hferentschik** [comment]
The verbosity level will chang with #634. This part of the issue will be handled there. Regarding the feedback after running the 'enable' command, we should print something to the console. This will be part of some other minor adjustments coming up.

**Comment by gbraad** [comment]
@praveenkumar basically does what I described in #563 in a less verbose way:

```
Entered function: AddSudoersRoleForUser("developer")

Running command: 'oc login -u system:admin' took: 23.207808285s
Running command: 'oc adm policy add-cluster-role-to-user sudoer developer' took: 199.723574ms
Running command: 'oc login -u developer' took: 23.009534518s
```

Please also mark of things that have been mentioned in #485.

**Comment by praveenkumar** [comment]
@gbraad yes agree on that.

**Comment by jorgemoralespou** [comment]
Similar feedback as here:
https://github.com/minishift/minishift/issues/634#issuecomment-291546740

**Comment by hferentschik** [comment]
Merged via pull request #761 which prints some additional enable/disable messages. This addresses the second point mentioned in https://github.com/minishift/minishift/issues/652#issue-219154139. The other points are covered by other issues.
